### DISCUSSION TITLE
SDKをアップグレードしたところアバターの一部が非表示にできなくなりました。

### DISCUSSION THREAD

**Discussion by papicham** [pixiv/vroid-sdk-developers#178] [Q&A] [answered]
度々の質問恐れ入ります。

質問内容を修正するためタイトルと異なってしまい申し訳ありません。
アウトラインを反映さる　Use Attach Mtoonのチェックを外すと綺麗にアバターを非表示にできるのですが、アウトラインを表示させたままアバターを綺麗に非表示にする方法はございますでしょうか。

ご教授いただけますと幸いです。

![非表示](https://github.com/user-attachments/assets/b0ec1861-c7ba-4be4-be2e-edd98840199c)

![BP](https://github.com/user-attachments/assets/c44b6bd4-ca20-42dd-b066-417f42bf1cea)

**Comment by mameoPix** [comment, marked as answer]
下図のようにBlueprintを組むことで表示/非表示の更新を行うことが出来ます。(図ではVRMのMeshを非表示に更新)
![image](https://github.com/user-attachments/assets/bf579ff8-5af1-402a-859e-9cbeda06f4fe)

また、Tick等で表示/非表示を切り替えるのであれば
BeginPlay等で初期化時にAttachRetargetMannequinActorで生成したAttachActorを変数に保存するなどしてキャッシュしておくと
より処理負荷の少ない処理を組むことが出来るようになると思います。

**Reply by papicham** [reply]
無事解決できました！
Tickに関するアドバイスもありがとうございます。
初心者なのですが沢山サポートして頂き、お陰様で
VroidHubを使えるシステムを凄く楽しませて頂いております。
いつも本当にありがとうございます。
